libs/java/server_common/src/main/java/com/yahoo/athenz/common/ServerCommonConsts.java

```diff
@@ -23,6 +23,24 @@
 
     public static final String USER_DOMAIN_PREFIX = "user.";
 
+    public static final String ZTS_PROP_AWS_REGION_NAME               = "athenz.zts.aws_region_name";
+    public static final String ZTS_PROP_AWS_CREDS_UPDATE_TIMEOUT      = "athenz.zts.aws_creds_update_timeout";
+    public static final int ZTS_PROP_AWS_CREDS_UPDATE_TIMEOUT_DEFAULT = 900;
+
+    // For S3ChangeLogStore
+    public static final String ZTS_PROP_AWS_BUCKET_NAME = "athenz.zts.aws_bucket_name";
+
+    // For ZMSFileChangeLogStore
+    public static final String ZTS_PROP_ZMS_URL_OVERRIDE    = "athenz.zts.zms_url";
+    public static final String ATHENZ_SYS_DOMAIN            = "sys.auth";
+    public static final String ZTS_SERVICE                  = "zts";
+    public static final String PROP_USER_DOMAIN             = "athenz.user_domain";
+    public static final String PROP_ATHENZ_CONF             = "athenz.athenz_conf";
+    public static final String ZTS_PROP_FILE_NAME           = "athenz.zts.prop_file";
+
+    // for tests
+    public static final String ZTS_PROP_AWS_PUBLIC_CERT = "athenz.zts.aws_public_cert";
```

Comment:
I don't think this should be here. The public cert is only used for validating instance documents so it belongs to ZTS server only.